security.md

```diff
@@ -0,0 +1,15 @@
+# Security
+
+Security is at the heart of a healthy and dependable software ecosystem, and therefore also at the heart of the SSWG. The documents linked from this page are intended to provide clear and simple guidance for both users and developers of packages listed in the SSWG's package index.
+
+## Available information
+
+The information we have is split into different scenarios, depending on why you are here. Each piece of information is supposed to be self-contained, so please select the most appropriate one. If the reason you are here is not covered, please get in touch with the [SSWG on the forums](https://forums.swift.org/c/server/serverdev/14). If you're just curious, feel free to read it all at once 😊.
```

Comment:
```suggestion
The information we have is split into different scenarios, depending on the use case. Each piece of information is supposed to be self-contained, so please select the most appropriate one. If the reason you are here is not covered, please get in touch with the [SSWG on the forums](https://forums.swift.org/c/server/serverdev/14). If you're just curious, feel free to read it all at once 😊.
```

Reply:
Thanks, wasn't a big fan of "use case" here, so reworded it to

```markdown
The information is split into different scenarios, depending on what brings you here. Each piece of information is supposed to be self-contained, so please select the most appropriate one. If your situation isn't covered, please get in touch with the [SSWG on the forums](https://forums.swift.org/c/server/serverdev/14).
```

Reply:
does that work for you?